packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -676,6 +676,19 @@ export const Table = <
     }
   }, [state, instance.filteredRows, onFilter]);
 
+  const lastPassedColumns: React.MutableRefObject<Column<T>[] | null> =
```

Comment:
`const lastPassedColumns = React.useRef([] as Column<T>[]);`

This looks a bit cleaner in my opinion but it's a nitpick

Reply:
**[DONE]** Thanks, done